cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/common/MeasureEvaluator.java

```diff
@@ -209,7 +210,7 @@ protected MeasureDef evaluate(MeasureDef measureDef, MeasureReportType type, Lis
                 type.toCode(),
                 subjectIds.size());
 
-        MeasureScoring scoring = measureDef.scoring();
```

Comment:
issue: the _new_ changes in MeasureEvaluator only have 50% coverage. The reason is that this line got flagged as changed. See the coverage report here:
https://app.codecov.io/gh/cqframework/clinical-reasoning/pull/391#bd8a68b9870c94fa5c632021c761e5fb-L214

Short version: Add a negative test that asserts the the Measure evaluation blow ups correctly when scoring is missing